### DISCUSSION TITLE
Is possible too see pinned album icon when shared have shared link on web version

### DISCUSSION THREAD

**Discussion by Tuscann** [ente-io/ente#2929] [Enhancements]
### Description

1. Open Web ente [https://web.ente.io/gallery](https://web.ente.io/gallery)
2. Create album da
3. Share album with date ti expired . Wait this date to come album to expire
4.Open album and click pin album
5. Album is not pinned (If you watch UI)
6. Possible to unpin the album

**Problem is same position for pinned album and share link icons** Bottom left.

Web version
![3333](https://github.com/user-attachments/assets/bfd1d25c-6fb3-4f49-a257-3d0ac375210b)

iOS version 

Share link icon is on top left of album and pinned icon is bottom right of album.

![image](https://github.com/user-attachments/assets/fff19d70-cb5e-4122-ae13-17053b22512f)

 

### What product are you using?

Ente Photos

### What platform are you using?

Web - Chrome

**Comment by ua741** [comment]
@Tuscann 
Currently, there are many differences between the UX of mobile and desktop. The team is aware of all of them.

We will focus on achieving complete parity between the two platforms next year. However, due to limited bandwidth, we are unable to prioritize these improvements right now, as there are other high-priority items. Until then, we kindly request that you avoid comparing the UX of mobile and desktop.
